.gitignore

```diff
@@ -50,3 +50,4 @@ out/
 **/mainGeneratedDataTemplate
 **/mainGeneratedRest
 gobblin-dist*
+gobblin.tar.gz
```

Comment:
Would you be opposed to adding these to the ignore list as well:
 * ParallelRunnerTest
 * GobblinHelixTaskTest
 * FsDatasetStateStoreTest/TestJob